src/app/core/services/auth.service.ts

```diff
@@ -60,15 +60,19 @@ export class AuthService {
     this.phaseService.reset();
     this.dataService.reset();
     this.githubEventService.reset();
-    this.titleService.setTitle(
-      require('../../../../package.json').name
-      .concat(' ')
-      .concat(require('../../../../package.json').version)
-    );
+    this.setTitle();
     this.issueService.setIssueTeamFilter('All Teams');
     this.reset();
   }
 
+  setTitle(): void {
```

Comment:
I think it'll be neater to locate both methods under the `AuthService`.
One method sets the title for the `Select Your Session` page, and the other sets the title once user has logged in.

Can name them `setLandingPageTitle` and `setTitleWithPhaseDetail` respectively (I'm open to other naming suggestions)

Reply:
I see, I think I misinterpreted the description in your issue! Will change accordingly.